src/lib/models/batch.js

```diff
@@ -33,7 +33,8 @@ const BATCH_ERROR_CODE = {
   failedToCreateBillRun: 50,
   failedToDeleteInvoice: 60,
   failedToProcessTwoPartTariff: 70,
-  failedToGetChargeModuleBillRunSummary: 80
+  failedToGetChargeModuleBillRunSummary: 80,
+  failedToProcessRebilling: 90
```

Comment:
This isn't something I want you to change - just making an observation: Those codes don't need to be codes. In the longer term, we might want to change those such that we simply store the name of the error type. E.g. Instead of `90` we would simply store `failedToProcessRebilling`.

Just a thought for the future. Not something to worry about in the context of this PR.

Reply:
True. 
I'd also like the UI to show the error then we wouldn't have to fish around in the DB.